addons/account/migrations/9.0.1.1/post-migration.py

```diff
@@ -336,6 +336,31 @@ def map_account_tax_template_type(cr):
         table='account_tax_template', write='sql')
 
 
+def migrate_account_sequence_fiscalyear(cr):
+    """ Migrate subsequences from fiscalyears to sequence date ranges
+    so that the next number is aligned with the last assigned number """
+    openupgrade.logged_query(
+        cr,
+        """ \
```

Comment:
Why the `\`?

Reply:
I *think* it looks more aligned in the logfile.